examples/asr/conf/citrinet/config_pt.yaml

```diff
@@ -0,0 +1,230 @@
+
+  spec_augment:
+    _target_: nemo.collections.asr.modules.SpectrogramAugmentation
+    freq_masks: 2
+    time_masks: 5
+    freq_width: 27
+    time_width: 0.05
+
+  encoder:
+    _target_: nemo.collections.asr.modules.ConvASREncoder
+    feat_in: *n_mels
+    activation: relu
+    conv_mask: true
+
+    jasper:
+      - filters: 128
+        repeat: 1
+        kernel: [11]
+        stride: [1]
+        dilation: [1]
+        dropout: *dropout
+        residual: true
+        separable: *separable
+        se: true
+        se_context_size: -1
+
+      - filters: 128
+        repeat: 1
+        kernel: [13]
+        stride: [2]  # *stride
+        dilation: [1]
+        dropout: *dropout
+        residual: true
+        separable: *separable
+        se: true
+        se_context_size: -1
+        stride_last: true
+        residual_mode: "stride_add"
+
+      - filters: 256
+        repeat: *repeat
+        kernel: [13]
+        stride: [1]
+        dilation: [1]
+        dropout: *dropout
+        residual: true
+        separable: *separable
+        se: true
+        se_context_size: -1
+
+      - filters: 256
+        repeat: *repeat
+        kernel: [15]
+        stride: [1]
+        dilation: [1]
+        dropout: *dropout
+        residual: true
+        separable: *separable
+        se: true
+        se_context_size: -1
+
+      - filters: 256
+        repeat: 1
+        kernel: [ 17 ]
+        stride: [ 2 ]  # *stride
+        dilation: [ 1 ]
+        dropout: *dropout
+        residual: true
+        separable: *separable
+        se: true
+        se_context_size: -1
+        stride_last: true
+        residual_mode: "stride_add"
+
+      - filters: 256
+        repeat: *repeat
+        kernel: [17]
+        stride: [1]
+        dilation: [1]
+        dropout: *dropout
+        residual: true
+        separable: *separable
+        se: true
+        se_context_size: -1
+
+      - filters: 256
+        repeat: *repeat
+        kernel: [19]
+        stride: [1]
+        dilation: [1]
+        dropout: *dropout
+        residual: true
+        separable: *separable
+        se: true
+        se_context_size: -1
+
+      - filters: 256
+        repeat: 1
+        kernel: [ 21 ]
+        stride: [ 2 ]  # *stride
+        dilation: [ 1 ]
+        dropout: *dropout
+        residual: true
+        separable: *separable
+        se: true
+        se_context_size: -1
+        stride_last: true
+        residual_mode: "stride_add"
+
+      - filters: 256
+        repeat: 1
+        kernel: [21]
+        stride: [1]
+        dilation: [1]
+        dropout: 0.0
+        residual: false
+        separable: *separable
+        se: true
+        se_context_size: -1
+
+      - filters: &enc_feat_out 1024
+        repeat: 1
+        kernel: [1]
+        stride: [1]
+        dilation: [1]
+        dropout: 0.0
+        residual: false
+        separable: *separable
+        se: true
+        se_context_size: -1
+
+  dec_out: &dec_out 128
```

Comment:
Is this a constant? Then it musst be at the top right after `model` scope begins or inside `model.model_defaults`. Look at contextnet configs for examples.